MatrixKit/Views/Authentication/MXKAuthenticationFallbackWebView.m

```diff
@@ -67,6 +76,15 @@ - (void)openFallbackPage:(NSString *)fallbackPage success:(void (^)(MXCredential
     activityIndicator.center = self.center;
     [self addSubview:activityIndicator];
     [activityIndicator startAnimating];
+
+    // Delete CAS OM cookies to prevent auto login
+    for(NSHTTPCookie *cookie in [[NSHTTPCookieStorage sharedHTTPCookieStorage] cookies])
+    {
+        if([[cookie domain] isEqualToString:@"cas.openmarket.com"])
```

Comment:
How about only cleaning them on explicit logout?  It's meant to be a 
feature that CAS remembers your SSO unless it either times out your 
session or you explicitly log out.

On 28/10/2015 10:46, manuroe wrote:
> In MatrixKit/Views/Authentication/MXKAuthenticationFallbackWebView.m
>
> <https://github.com/matrix-org/matrix-ios-kit/pull/21#discussion_r43239950>:
>
>
>> @@ -67,6 +76,15 @@ - (void)openFallbackPage:(NSString
>> *)fallbackPage success:(void (^)(MXCredential
>> activityIndicator.center = self.center; [self
>> addSubview:activityIndicator]; [activityIndicator startAnimating];
>> + +    // Delete CAS OM cookies to prevent auto login +
>> for(NSHTTPCookie *cookie in [[NSHTTPCookieStorage
>> sharedHTTPCookieStorage] cookies]) +    { +        if([[cookie
>> domain] isEqualToString:@"cas.openmarket.com"])
>
> I am not comfortable with putting a special handling on a particular
>  page url. Can we clean cookies every time?
>
> — Reply to this email directly or view it on GitHub
> <https://github.com/matrix-org/matrix-ios-kit/pull/21/files#r43239950>.
>